src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2844,7 +2844,18 @@ private string GetStaticResourceName(XamlMemberDefinition member, INamedTypeSymb
 				// If the property Type is attributed with the CreateFromStringAttribute
 				if (IsXamlTypeConverter(targetPropertyType))
 				{
-					var memberValue = $"StaticResources.{SanitizeResourceName(resourcePath)}";
+					string memberValue;
+
+					// Build the member string based on wether it's a local
+					// StaticResource or a global StaticResource
+					if (_staticResources.TryGetValue(resourcePath, out var _))
```

Comment:
@davidjohnoliver  I'm unsure what the purpose of `_isGeneratingGlobalResource ` is in the context of this function, but in my case it was always false, therefore I determine a global StaticResource using the `TryGetValue` on `_staticResources`.

Reply:
`_isGeneratingGlobalResource` just means that the Xaml being generated is either a `<ResourceDictionary>` file, or `App.xaml`

There's no such thing as a 'local' StaticResource in the current version of Uno, it's just that the generation logic is different for locally-defined resources.

Reply:
ok, @davidjohnoliver  thanks for clarifying!